openssl/goopenssl.h

```diff
@@ -65,11 +68,25 @@ FOR_ALL_OPENSSL_FUNCTIONS
 #undef DEFINEFUNC_RENAMED
 #undef DEFINEFUNC_FALLBACK
 
-// This wrapper allocate out_len on the C stack, and check that it matches the expected
+// These wrappers allocate out_len on the C stack, and check that it matches the expected
```

Comment:
How do these wrappers actually check the value was what was expected?

Reply:
Good catch, I took the wrapping idea from the [boring branch](https://github.com/golang/go/blob/dev.boringcrypto.go1.17/src/crypto/internal/boring/aes.go#L19) and just copied the comment as is. 

We use these wrappers in functions where we can't know which is the expected output length, so I will reword the comment.